migrations/1589468201229-add_more_school_years.js

```diff
@@ -0,0 +1,47 @@
+const { info, warning } = require('../src/logger');
+
+const { connect, close } = require('../src/utils/database');
+const { yearModel: Year } = require('../src/services/school/model');
```

Comment:
Please define the model here, because in the future it can be changed and not fitting to this migration anymore

Reply:
done